### DISCUSSION TITLE
Don't required experimental "packageManager" feature

### DISCUSSION THREAD

**Discussion by jbisotti** [Siouan/frontend-gradle-plugin#236] [Ideas]
I can see this being a non-starter for a lot of projects. Consider just specifying the version of Yarn (for example) along-side the NPM version, like the `maven-frontend-plugin` does.

**Comment by v1nc3n4** [comment]
Hello,

Thank you for your suggestion.
I don't understand the reason of this one though.
Can you explain what are the pain points you may have encountered using this plugin?

BR

**Comment by v1nc3n4** [comment]
Hello @jbisotti,

I'd like to improve my previous answer. Before deciding to leave the management of the package manager to Corepack, I had a [discussion](https://github.com/yarnpkg/berry/discussions/4628) on July 2022 with the Yarn team about the stability and production-ready status of Corepack. Shortly, Corepack is production-ready, and this is the official way to install Yarn. There are many other discussions about stability of Corepack and the remove of the "experimental" flag in some places, since 2022. Corepack is used more and more because it simplifies management of the package manager. For example:

- https://github.com/nodejs/corepack/issues/104
- https://github.com/nodejs/node/issues/50963

At the moment, if you don't want your project to rely on Corepack due to its "experimental" status, the only solution with this plugin and Yarn 1/2+ is to use release 6.0.0.

BR
